interfaces/testType.ts

```diff
@@ -1,3 +1,5 @@
 export default interface ITest {
+  headerChildren: JSX.Element;
```

Comment:
This type only for testId prop. It should be better extends from ITest and add another props in the component